src/pages/en/editor-setup.md

```diff
@@ -37,3 +37,39 @@ In addition to local editors, Astro also runs well on in-browser hosted editors,
 - [StackBlitz](https://stackblitz.com/) and [CodeSandbox](https://codesandbox.io/) - online editors that run in your browser, with built-in syntax highlighting support for `.astro` files. No installation or configuration required!
 - [GitHub.dev](https://github.dev/) - allows you to install the Astro VS Code extension as a [web extension](https://code.visualstudio.com/api/extension-guides/web-extensions), which gives you access to only some of the full extension features. Currently, only syntax highlighting is supported.
 - [Gitpod](https://gitpod.io/) - a full dev environment in the cloud that can install the official Astro VS Code Extension from Open VSX.
+
+## Other tools
+
+### ESLint
+
+[ESLint](https://eslint.org/) is a popular linter for JavaScript and JSX. For Astro support, [a community maintained plugin](https://github.com/ota-meshi/eslint-plugin-astro) can be installed
+
+See [the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/user-guide/) for more information on how to install and setup this for your project
+
+### Prettier
+
+[Prettier](https://prettier.io/) is a popular formatter for JavaScript, HTML, CSS and more. To add Astro support (`.astro` files), we created and maintain an [official Prettier Astro plugin](https://github.com/withastro/prettier-plugin-astro)
+
+To get started, first install Prettier and the plugin:
+
+```shell
+npm install --save-dev prettier prettier-plugin-astro
+```
+
+Prettier will then automatically detect the plugin and use it to process `.astro` files when you run it:
+
+```shell
+prettier --write .
+```
+
+For more information about the supported options, how to get it setup when using Prettier inside VS Code and more, see [the plugin's README](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md)
```

Comment:
I usually break up or use one word for “set up” depending on how it’s used:

`setup` = noun
`set up` = verb + preposition

I think that’s right anyway 😅 — perhaps @sarah11918 can confirm.

```suggestion
For more information about the supported options, how to get it set up when using Prettier inside VS Code and more, see [the plugin's README](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md).
```

Reply:
Looked on the internet and `set up` seems to be right here!

Reply:
Yes, Chris is correct. `setup` is a noun or adjective, otherwise, use `set up` as a verb. 

It would be a "crisper" sentence if we could use "set up" more directly, so maybe something like:

See the [Prettier plugin's README](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md) for more information about its supported options, how to set up Prettier inside VS Code, and more.

(Just swapped the order to avoid both the Oxford comma, and then the comma after "and more." We do often use "See this for... " "Check out this for..." and "Read about this..." sentence structure throughout docs, too.)